docker/centos7/Dockerfile

```diff
@@ -378,11 +378,18 @@ RUN curl -Ls https://archive.apache.org/dist/maven/maven-3/3.8.3/binaries/apache
     echo '[ -x /opt/maven/bin/mvn ] && export PATH=/opt/maven/bin/:$PATH' >> /etc/profile.d/maven.sh && \
     rm -rf /tmp/*
 
-# install rocksdb to /opt
+# extract rocksdb 7 source to /opt
 RUN curl -Ls https://github.com/facebook/rocksdb/archive/refs/tags/v7.10.2.tar.gz -o rocksdb.tar.gz && \
     echo "4619ae7308cd3d11cdd36f0bfad3fb03a1ad399ca333f192b77b6b95b08e2f78 rocksdb.tar.gz" > rocksdb-sha.txt && \
     sha256sum --quiet -c rocksdb-sha.txt && \
     tar --directory /opt -xf rocksdb.tar.gz && \
+    rm -rf /tmp/* \
+
+# extract rocksdb 8 source to /opt
```

Comment:
Don't we need to change cmake to use rocksdb 8.1 ?

Reply:
yes, the cmake changes for `main` and `release-7.3` are already in. 

`release-7.1` and `release-7.2` have not been updated.

Reply:
I was just confused seeing the cmc_ci still using rocksdb-7.10 version